packages/datepicker/src/react/__stories__/index.story.tsx

```diff
@@ -255,7 +253,10 @@ export const RangeDateCalendar: Story = () => {
           return (
             <button
               {...renderProps}
-              {...isInRange(dateObj.date)}
+              className={classNames(
+                renderProps.className,
+                isInRange(dateObj.date)
+              )}
```

Comment:
New merge required because of change in how we attach selectors to the element (need composed in className attribute).